components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/BrowserAction.kt

```diff
@@ -21,8 +20,7 @@ import android.graphics.drawable.Drawable
 data class BrowserAction(
     val title: String,
     val enabled: Boolean,
-    val icon: Drawable,
-    val uri: String,
+    val loadIcon: suspend (Int) -> Bitmap?,
```

Comment:
Is this related to AMO add-on icon?

Reply:
This icon is specified by extension itself either in the manifest or via `setIcon`:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setIcon